recipes.json

```diff
@@ -1,43 +1,109 @@
 {
     "carbonara": {
         "ingredients": {
-            "egg": 2,
-            "parmesan cheese": 1,
-            "spaghetti": 1,
-            "bacon": 4
+            "egg": [
+                2,
+                "unit"
+            ],
+            "parmesan cheese": [
+                200,
+                "Gramms"
+            ],
+            "spaghetti": [
+                1,
+                "Gramms"
+            ],
+            "bacon": [
+                4,
+                "Rashers"
+            ]
         },
         "portions": 2
     },
     "bacon and egg muffins": {
         "ingredients": {
-            "egg": 1,
-            "english muffins": 1,
-            "bacon": 2,
-            "butter": 1
+            "egg": [
+                1,
+                "Unit"
+            ],
+            "english muffins": [
+                1,
+                "Unit"
+            ],
+            "bacon": [
+                2,
+                "Rashers"
+            ],
+            "butter": [
+                1,
+                "tsp"
```

Comment:
should this be capitalised?

Reply:
i didnt think it was for teaspoon. i thought it would be for Tablespoon.....i could be wrong though (more than likely)